### ISSUE TITLE
Bug: extension doesn't load with latest version of Automatic1111 / gradio

### ISSUE THREAD

**Issue by 2x-y** [Akegarasu/sd-webui-model-converter#9]
I think the gradio version has been bumped and is causing an issue when initialising the UI.

```
Error executing callback ui_tabs_callback for H:\GitHub\stable-diffusion-webui\extensions\sd-webui-model-converter\scripts\convert.py
Traceback (most recent call last):
  File "H:\GitHub\stable-diffusion-webui\modules\script_callbacks.py", line 125, in ui_tabs_callback
    res += c.callback() or []
  File "H:\GitHub\stable-diffusion-webui\extensions\sd-webui-model-converter\scripts\convert.py", line 94, in add_tab
    model_converter_convert.click(
  File "H:\GitHub\stable-diffusion-webui\venv\lib\site-packages\gradio\events.py", line 129, in __call__
    dep, dep_index = self.trigger.set_event_trigger(
  File "H:\GitHub\stable-diffusion-webui\venv\lib\site-packages\gradio\blocks.py", line 195, in set_event_trigger
    check_function_inputs_match(fn, inputs, inputs_as_dict)
  File "H:\GitHub\stable-diffusion-webui\venv\lib\site-packages\gradio\utils.py", line 940, in check_function_inputs_match
    if not is_special_typed_parameter(name):
  File "H:\GitHub\stable-diffusion-webui\venv\lib\site-packages\gradio\utils.py", line 929, in is_special_typed_parameter
    is_event_data = issubclass(parameter_types.get(name, int), EventData)
TypeError: issubclass() arg 1 must be a class
```

Extension version: 7a998ede

Automatic1111 Info:
python: 3.10.6
torch: 1.13.1+cu117
xformers: 0.0.16rc425
gradio: 3.23.0
commit: 91ae48fd
checkpoint: cc6cb27103

**Comment by Harvester62** [comment]
I confirm, getting the same error messages on Windows 10 Pro, Nvidia GTX1070 (8GB VRAM)
A1111 SD Web UI info:
python: 3.10.6  
torch: 1.13.1+cu117  
xformers: 0.0.17.dev464  
gradio: 3.23.0  
commit: [91ae48fd]
checkpoint: [9012b038a9]

**Comment by LieDeath** [comment]
absolutely the same, confirm gradio 3.23.0 problem.

Error executing callback ui_tabs_callback for D:\AI6\stable-diffusion-webui\extensions\sd-webui-model-converter\scripts\convert.py
Traceback (most recent call last):
  File "D:\AI6\stable-diffusion-webui\modules\script_callbacks.py", line 125, in ui_tabs_callback
    res += c.callback() or []
  File "D:\AI6\stable-diffusion-webui\extensions\sd-webui-model-converter\scripts\convert.py", line 94, in add_tab
    model_converter_convert.click(
  File "D:\AI6\stable-diffusion-webui\venv\lib\site-packages\gradio\events.py", line 129, in __call__
    dep, dep_index = self.trigger.set_event_trigger(
  File "D:\AI6\stable-diffusion-webui\venv\lib\site-packages\gradio\blocks.py", line 195, in set_event_trigger
    check_function_inputs_match(fn, inputs, inputs_as_dict)
  File "D:\AI6\stable-diffusion-webui\venv\lib\site-packages\gradio\utils.py", line 940, in check_function_inputs_match
    if not is_special_typed_parameter(name):
  File "D:\AI6\stable-diffusion-webui\venv\lib\site-packages\gradio\utils.py", line 929, in is_special_typed_parameter
    is_event_data = issubclass(parameter_types.get(name, int), EventData)
TypeError: issubclass() arg 1 must be a class

**Comment by 2x-y** [comment]
Fixed by commit d19e281
